packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,56 @@
+
+%global repo_orgname RedHatInsights
+%global repo_name insights-client-role
+%global repo_version 1.5
+
+%global role_orgname %{repo_orgname}
+%global role_name insights-client
+%global role_version %{repo_version}
+
+Name: ansiblerole-insights-client
+Summary: Packaging of the insights-client Ansible role
+Version: %{role_version}
+Release: 1%{?dist}
+
+License: ASL 2.0
+
+Source: https://github.com/%{repo_orgname}/%{repo_name}/archive/v%{repo_version}.tar.gz#/%{role_name}-%{role_version}.tar.gz
+
+Url: https://github.com/%{repo_orgname}/%{repo_name}/
+BuildArch: noarch
+
+Requires: ansible
+
+%description
+This package installs the insights-client Ansibile role.
+
+Make sure that "/usr/share/ansible/roles" is on your Ansible role_path.
+
+%prep
+%setup -qc
+%build
+%install
+mkdir -p $RPM_BUILD_ROOT%{_datadir}/ansible/roles
+
+cp -pR %{repo_name}-%{role_version} $RPM_BUILD_ROOT%{_datadir}/ansible/roles/%{role_orgname}.%{role_name}
+
+mkdir -p $RPM_BUILD_ROOT%{_pkgdocdir}/
+
+cp -p $RPM_BUILD_ROOT%{_datadir}/ansible/roles/%{role_orgname}.%{role_name}/README.md \
+      $RPM_BUILD_ROOT%{_datadir}/ansible/roles/%{role_orgname}.%{role_name}/LICENSE \
```

Comment:
I'm not sure this is needed if you already add it via ```%license```.

Reply:
RPM has one convention for where to put package Licence, README, and examples, and Ansible Roles packages have a different convention for where to put them.   

Rather than honor one convention over the other, i choose to install the files into both places.  I'd rather leave it as is, unless someone can thing of a problem this causes.